tests/cookbook_cpo_olivine.prm

```diff
@@ -0,0 +1,3 @@
+# input file for cpo olivine cookbook
+
+include $ASPECT_SOURCE_DIR/cookbooks/crystal_preferred_orientation_olivine_fraters_billen_2021/olivineA.prm
```

Comment:
How expensive is this model? It looks like you compute 600 timesteps in the test, maybe you can reduce the end time to only compute 10 or so for the test? This would also make the test output smaller.